src/app/components/LinkedData/index.tsx

```diff
@@ -111,12 +111,32 @@ const LinkedData = ({
 
   const hasByline = !!bylineLinkedData;
 
-  const { authorName, authorTopicUrl, twitterLink, authorImage, location } =
-    bylineLinkedData || {};
+  const places: (string | undefined)[] = [];
```

Comment:
Let's remove undefined, and filter out any null/undefined values from the places array instead
```suggestion
  const places: string[] = [];
```